docs/remote-vm-setup.md

```diff
@@ -0,0 +1,127 @@
+# Running Scripts on a Remote VM
+
+This guide explains how to set up and run eBPF for Windows CI/CD scripts on a remote VM.
+
+---
+
+## 1. Prepare the Remote VM
```

Comment:
Suggest: Is it possible to create a new script, setup_remote_vm.ps1 (or whatever name makes sense), and then add all of these into the script?